code/modules/antagonists/blob/blob_mobs.dm

```diff
@@ -93,12 +93,25 @@
 	var/mob/living/carbon/human/oldguy
 	var/is_zombie = 0
 	gold_core_spawnable = HOSTILE_SPAWN
+	var/list/disease = list()
 
 /mob/living/simple_animal/hostile/blob/blobspore/Initialize(mapload, var/obj/structure/blob/factory/linked_node)
 	if(istype(linked_node))
 		factory = linked_node
 		factory.spores += src
 	. = ..()
+	/*var/datum/disease/advance/random/blob/R = new //either viro is cooperating with xenobio, or a blob has spawned and the round is probably over sooner than they can make a virus for this
```

Comment:
Noooo!!!! You cant just have commented code no!!!! 😳

Reply:
this shit didn't work, and i couldnt figure out how to fix it. Blob spores seemed to fuck getdiseaseid when spawned in this way, spawning cultures that would runtime and not work. Currently, blob spores have no disease by default, but contain the disease that spawned them if spawned by the blobspore symptom